### ISSUE TITLE
Broken page display on iPad

### ISSUE THREAD

**Issue by sa31fh** [swash99/ims#113]
Saw this issue on their iPad (using Google Chrome):
The two sections on category_status page (i.e. categories column on the left, and the inventory entry page which occupies the right 3/4th of the page) is broken up. Instead of being on the right side, the inventory entry section is completely moved to below the categories column, so you have to scroll down to get to the entry table. 

Not sure how much you can do with just this description, but let me know if you want further details which could be helpful.

**Comment by swash99** [comment]
Added a fix for the issue. Atleast it wont shift down, hopefully.

Was this in portrait mode where you saw the issue or landscape?. If I've assumed the correct ipad dimensions the current layout should be fine in landspace but would be quite squeezed in portrait view so best to avoid

**Comment by sa31fh** [comment]
- I'll verify if your fix solved the problem next time I drop by there or talk to them. 
- This problem was in portrait mode. I can also suggest to use landscape mode which might be better for use but at least the site should not appear to be broken -- squeezed is fine, more reason for them to use in landscape mode. I actually don't see this issue on my phone which has an even smaller screen so I was a bit surprised when they showed me the problem.

**Comment by swash99** [comment]
Adjusted styles and add '@media' query for smaller screens.

**Comment by sa31fh** [comment]
Unfortunately this bug is still showing up on the iPad -- reopening this issue.
I'll try to get some more details to see if changing to landscape mode or using another browser fixes things. Will also ask to see if this can be tested on iPhone. It works on my small Android phone with no problems.

Lets keep this in the backlog for now. No need to focus too much on this.

**Comment by sa31fh** [comment]
Managed to experiment a little with devices and here is the result:
- The issue of the broken page is seen in both Google Chrome and Safari browsers.
- The issue is seen on both iPad and iPhone. However it does work properly on Mac desktop.
- Switching to landscape mode does not fix the issue. 

My conclusions from what I observed:
- I don't think this is an issue of insufficient horizontal space on the screen since the problem persists in landscape mode. 
- This seems to be an iOS specific thing.

Possible paths to pursue:
- Search for broken layout issue specifically related to how you've implemented things. For example, "divs are displayed vertically instead of side by side" etc.
- Search for iOS (iPad/iPhone) specific issues for broken layout.
- Try to get your hand on other devices and see if the site appears broken there as well e.g. android devices, blackberry devices, any tablets, different OS's and sizes -- just to see if the issue is reproducible outside iPad and iPhone.
